### ISSUE TITLE
WCAG 3.3.1: identify invalid inputs with `aria-invalid`

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#1032]
At the moment, some answer inputs with invalid values have the `has-error` class. This can't be read by assistive tech.
The [`aria-invalid`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid) attribute should be set as well.
